### ISSUE TITLE
A11y attribute tweaks

### ISSUE THREAD

**Issue by EdwinGuzman** [NYPL-Simplified/web-reader-landing-page#185]
From @wlla : 
* Facebook logo has incorrect `aria-labelledby attribute` since currently reads as "Brooklyn Public Library logo" on Home and Genre pages. Attribute is correct in About page version of same element.
8 Update titles in Book detail pages to differentiate from book text in web reader by appending "Item detail page" to the current title. Example follows. 
Current: `<title>Subway Library | Sense and Sensibility</title>`
Update to: `<title>Subway Library | Sense and Sensibility Item detail page</title>`
* `Aria-labelled-by` attribute for Transit Wi-Fi in footer on About page should be updated to 'transit-wi-fi'
* In Genre and Home pages, footer svgs need aria-labelled-by or what not attributes. Crib from About page.
* On about page: Logo and svgs need to be relabeled so that "MTA" is "M.T.A" and "WiFi" is "Wi-Fi"
[Book detail] (and possibly other pages)
* In email share svg
id = email
Current: aria-labelledby="email"
Update to: aria-labelledby="share-via-email"
